src/sonic-config-engine/sonic-cfggen

```diff
@@ -137,6 +137,13 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+def load_namespace_config(asic_name):
+    if not SonicDBConfig.isInit():
```

Comment:
>isInit [](http://example.com/codeflow?start=25&length=6)

You should check `isInit` or `isGlobalInit` based on single/multi-asic.

Reply:
Yes. Fixed.